catalog/plugin/catalog-plugin-videothumbnail/src/main/java/org/codice/ddf/catalog/content/plugin/video/VideoThumbnailPlugin.java

```diff
@@ -175,27 +184,29 @@ private boolean isVideo(final ContentItem contentItem) {
     private void createThumbnail(final ContentItem contentItem) throws PluginExecutionException {
         LOGGER.debug("About to create video thumbnail.");
 
+        Path contentPath = Paths.get(tempDirectory.toAbsolutePath()
+                .toString(), contentItem.getFilename());
         try {
+
+            Files.copy(contentItem.getInputStream(), contentPath);
```

Comment:
It's an implementation detail that our default `ContentItemImpl` is backed by the file system. Callers won't know if that's the case, or if content is coming from some other storage mechanism (RDBMS, HDFS, etc.). It is inefficient to copy this file that already exists but that's not always going to be the case.